migrate.py

```diff
@@ -217,32 +246,36 @@ def visit_Assign(self, node):
             continue
 
         # TODO what if it's in a different namespace (different spec)? do we care?
-        new_fragment = '%s.%s.%s' % (args.namespace, fragment_collection, fragment)
-        # TODO make sure to replace only in DOCUMENTATION
-        plugin_data = plugin_data.replace(fragment, new_fragment)
+        new_fragment = f'{namespace}.{fragment_collection}.{fragment}'
+
+        # TODO figure out whether this can be improved
+        doc_val.value = doc_str_tmpl.format(
+            str_val=doc_txt.replace(fragment, new_fragment),
```

Comment:
shoudl we limit replacements to 1 instance to avoid the 'multipe things rewriten'?

Reply:
What do you mean?

Reply:
I guess you mean
```suggestion
            str_val=doc_txt.replace(fragment, new_fragment, count=1),
```
But I think that there's a few things to replace there. Anyway, it worked like this before so I'd just keep it until we hit some corner case.

Reply:
Should we ignore this for now?